python/src/opendp/context.py

```diff
@@ -866,19 +941,27 @@ def __call__(self, v):
         """Returns the transformation or measurement with the given parameter."""
         return self.partial(v)
 
-    def fix(self, d_in: Union[float, Sequence[Bound]], d_out: float, output_measure: Optional[Measure] = None, T=None):
+    def fix(
+        self, 
+        d_in: Union[float, Sequence[Bound]], 
+        d_out: Union[float, tuple[float, float]], 
+        output_measure: Optional[Measure] = None, 
+        bounds: Optional[tuple[float, float]] = None, 
+        T=None
+    ):
         """Returns the closest transformation or measurement that satisfies the given stability or privacy constraint.
 
         The discovered parameter is assigned to the param attribute of the returned transformation or measurement.
 
-        :param d_in:
-        :param d_out:
-        :param output_measure:
-        :param T:
+        :param d_in: Upper bound on the distance between adjacent input datasets.
+        :param d_out: Upper bound on the distance between adjacent output distributions.
+        :param output_measure: How to measure distances between output distributions.
+        :param bounds: The bounds for the parameter search.
+        :param T: The type of the parameter to search for.
         """
         # When the output measure corresponds to approx-DP, only optimize the epsilon parameter.
         # The delta parameter should be fixed in _cast_measure, and if not, then the search will be impossible here anyways.
-        if output_measure == fixed_smoothed_max_divergence():
+        if output_measure is not None and isinstance(output_measure.type, RuntimeType) and output_measure.type.origin == "Approximate":
```

Comment:
I worry about logic that depends on literal strings: typos can be easy to overlook.